mosdepth.nim

```diff
@@ -667,7 +732,7 @@ when(isMainModule):
   when not defined(release) and not defined(lto):
     stderr.write_line "[mosdepth] WARNING: built in debug mode; will be slow"
 
-  let version = "mosdepth 0.2.6"
+  let version = "mosdepth 0.2.5"
```

Comment:
make sure you rebase on top of current master so you don't do stuff like lower the version.

Reply:
Fixed